.gitignore

```diff
@@ -9,4 +9,7 @@ dependency-reduced-pom.xml
 *.ipr
 .idea/
 
-
+# ---- Eclipse
```

Comment:
In general, I recommend putting editor-specific ignores in the global gitignore on your device. It keeps the code's ignore clean as putting ignores for every possible editor would get messy quickly.

Reply:
Yeah, they probably should be removed too, IMHO. I'm a little picky personally, because once you start allowing those kinds of ignores, you'll need to also allow extra lines for Netbeans, Visual Studio Code, Sublime Text, etc.